src/pages/Editor.vue

```diff
@@ -197,10 +210,19 @@ export default {
       dialogDescription: "", // description for the dialog box
       dialogConfirmButtonConfig: {}, // config for the confirm button of the dialog box
       dialogCancelButtonConfig: {}, // config for the cancel button of the dialog box
+      dialogAction: "",
       hasUnpublishedChanges: false,
       // whether there are changes which have not been published
       // once plio is published, we don't automatically save changes
       // this tracks if there are unpublished changes
+      addItemIconConfig: { // config for icon of add item button
+        'enabled': true,
+        'iconName': 'plus-solid',
+        'iconClass': 'text-white h-5 w-5 mr-3'
+      },
+      addItemTitleConfig: { // config for title of add item button
+        'value': 'Add a question'
+      },
```

Comment:
Since we are specifically saying add question - better to name all variables related to it as question instead of item? We'll have separate buttons for other item types

Reply:
I dont think we should/need to do this abhi.
it will be a big change (matlab very small small changes overall)

and its not super urgent to do right now. Its not like what we have written is wrong. When we get to more item types, we'll parametrise then.
Skipping this for now

I will do one thing though, will add a comment everywhere where "question" is used in an "item" waala scope ki only "question" types are supported right now